bennu-io/src/main/dml/bennu-io.dml

```diff
@@ -98,7 +104,7 @@ relation FileStorages {
 	protected FileSupport playsRole fileSupport {
         multiplicity 1..1;
 	}
-	protected FileStorage playsRole fileStorage {
+	FileStorage playsRole fileStorage {
```

Comment:
why is this no longer protected?

Reply:
In the DML file the part of the relation was marked as protected. But then the base code was overridden to expose the fileStorageSet:
`-    @Override
-    public Set<FileStorage> getFileStorageSet() {
-        //FIXME: remove when the framework enables read-only slots
-        return super.getFileStorageSet();
-    }
-`
Thus defeating the purpose of protecting the underlying set. While this will in fact keep the from generating public methods addFileStorage() and removeFileStorage(); since the underlying set is exposed, elements may be added and removed from the set directly. So I don't see the purpose of purpose of protecting the set in this way.
Alternatively one could wrap the returnned set from the overrinden method, but that would introduce a significant overhead.
This way we don't have a false sense of security, and end up with less code.